### PR TITLE
Remove All Unused Files

### PR DESCRIPTION
**Pull request by dvaldivia**
Moves to controller model. Removes all un-used files.

### CODE REVIEW

File: cluster/config.go [hunk 1]
```diff
@@ -26,6 +26,10 @@ import (
 	"github.com/minio/minio/pkg/env"
 )
 
+func getNs() string {
```
Comment (cesnietor) [inline cluster/config.go:27]: sneaky

File: cluster/config.go [hunk 2]
```diff
@@ -26,6 +26,10 @@ import (
...
+	return "default"
```
Comment (cesnietor) [inline cluster/config.go:28]: are we planning to make this configurable from an env variable? or what was the reason of this function?

File: cluster/config.go (hunk 1 above)
Comment (dvaldivia) [reply cluster/config.go:27]: since we are moving to controller model, i'm introducing the controller

File: cluster/config.go (hunk 2 above)
Comment (dvaldivia) [reply cluster/config.go:28]: yes, this is still work in progress

File: cluster/crds/zones.go [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+										//									},
+										//								},
+										//							},
+										//						},
+										//					},
+										//				},
+										//			},
+										//		},
+										//	},
+										//},
```
Comment (cesnietor) [inline cluster/crds/zones.go]:
```suggestion
										//"node_template": {
										//	Type: "object",
										//	Properties: map[string]apiextensionsv1.JSONSchemaProps{
										//		"volumes": {
										//			Type: "array",
										//			Items: &apiextensionsv1.JSONSchemaPropsOrArray{
										//				Schema: &apiextensionsv1.JSONSchemaProps{
										//					Type: "object",
										//					Properties: map[string]apiextensionsv1.JSONSchemaProps{
										//						"metadata": {
										//							Type: "object",
										//							AdditionalProperties: &apiextensionsv1.JSONSchemaPropsOrBool{
										//								Allows: true,
										//							},
										//						},
										//						"volumes": {
										//							Type: "array",
										//							Items: &apiextensionsv1.JSONSchemaPropsOrArray{
										//								Schema: &apiextensionsv1.JSONSchemaProps{
										//									Type: "object",
										//									Properties: map[string]apiextensionsv1.JSONSchemaProps{
										//										"mount_path": {
										//											Type: "string",
										//										},
										//										"volume_type": {
										//											Type: "string",
										//										},
										//									},
										//									Required: []string{
										//										"volume_type",
										//									},
										//								},
										//							},
										//						},
										//					},
										//				},
										//			},
										//		},
										//	},
										//},
```
```suggestion
										//"node_template": {
										//	Type: "object",
										//	Properties: map[string]apiextensionsv1.JSONSchemaProps{
										//		"volumes": {
										//			Type: "array",
										//			Items: &apiextensionsv1.JSONSchemaPropsOrArray{
										//				Schema: &apiextensionsv1.JSONSchemaProps{
										//					Type: "object",
										//					Properties: map[string]apiextensionsv1.JSONSchemaProps{
										//						"metadata": {
										//							Type: "object",
										//							AdditionalProperties: &apiextensionsv1.JSONSchemaPropsOrBool{
										//								Allows: true,
										//							},
										//						},
										//						"volumes": {
										//							Type: "array",
										//							Items: &apiextensionsv1.JSONSchemaPropsOrArray{
										//								Schema: &apiextensionsv1.JSONSchemaProps{
										//									Type: "object",
										//									Properties: map[string]apiextensionsv1.JSONSchemaProps{
										//										"mount_path": {
										//											Type: "string",
										//										},
										//										"volume_type": {
										//											Type: "string",
										//										},
										//									},
										//									Required: []string{
										//										"volume_type",
										//									},
										//								},
										//							},
										//						},
										//					},
										//				},
										//			},
										//		},
										//	},
										//},
```
remove please

Comment (cesnietor) [inline cluster/crds/zones.go]:
```suggestion
```
remove please

**Review by cesnietor — CHANGES_REQUESTED** [review]
Can you please add how to test this.
Also README update?

**Review by cesnietor — CHANGES_REQUESTED** [review]
Can you please add how to test this.
Also README update?

File: cluster/crds/cluster.go [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package crds
+
+import (
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
```
Comment (cesnietor) [inline cluster/crds/cluster.go:23]: Add  please documentation. also to the whole definition if necessary to understand each of the properties.

Comment (dvaldivia) [reply cluster/crds/cluster.go:23]: it's on the readme now

**Comment by dvaldivia** [comment]
@cesnietor README updated

**Review by cesnietor — APPROVED** [review]
tested LGTM

**Pull request merged**